platform/x11/os_x11.cpp

```diff
@@ -267,6 +267,7 @@ Error OS_X11::initialize(const VideoMode &p_desired, int p_video_driver, int p_a
 
 // maybe contextgl wants to be in charge of creating the window
 #if defined(OPENGL_ENABLED)
+	setenv("DRI_PRIME", "1", 0);
```

Comment:
There should be a short comment explaining what this does.

Reply:
Comment added